generator/template/model_template.go

```diff
@@ -106,19 +108,13 @@ func getTableModelImports(modelType TableModel, tableMetaData metadata.Table) []
 	importPaths := map[string]bool{}
 	for _, columnMetaData := range tableMetaData.Columns {
 		field := modelType.Field(columnMetaData)
-		importPath := field.Type.ImportPath
-
-		if importPath != "" {
-			importPaths[importPath] = true
+		for _, importPath := range append([]string{field.Type.ImportPath}, field.Type.AdditionalImportPaths...) {
+			if importPath != "" {
+				importPaths[importPath] = true
+			}
 		}
 	}
-
-	var ret []string
-	for importPath := range importPaths {
-		ret = append(ret, importPath)
-	}
-
-	return ret
+	return slices.Collect(maps.Keys(importPaths))
```

Comment:
At the moment jet requires go version 1.22 or higher. Since `slices.Collect` and `maps.Keys` are added in version 1.23 build is failing. I like to keep minimal version couple version bellow latest go version not to force go upgrade on jet users.
To fix a build  `slices.Collect(maps.Keys(importPaths))` can be implemented as a helper function.